db/schema.rb

```diff
@@ -10,36 +10,38 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_06_02_141034) do
+ActiveRecord::Schema[7.0].define(version: 2022_06_09_235801) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "comments", force: :cascade do |t|
     t.text "text"
+    t.bigint "users_id", null: false
+    t.bigint "posts_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "author_id", null: false
-    t.bigint "post_id", null: false
-    t.index ["author_id"], name: "index_comments_on_author_id"
-    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["posts_id"], name: "index_comments_on_posts_id"
+    t.index ["users_id"], name: "index_comments_on_users_id"
   end
 
   create_table "likes", force: :cascade do |t|
+    t.bigint "users_id", null: false
+    t.bigint "posts_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "author_id", null: false
-    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["posts_id"], name: "index_likes_on_posts_id"
+    t.index ["users_id"], name: "index_likes_on_users_id"
   end
 
   create_table "posts", force: :cascade do |t|
     t.string "title"
```

Comment:
## OPTIONAL

- [ ] Kindly make sure you use the `Rails` names in convention since this is very important, sometimes it can even break your code if they are not respected. kindly note that all columns that carry references to other tables (associate) are always in singular example use `users_id`and `post_id` instead of `posts_id` and `user_id`.